Update overview + reorganize Rust tutorial (#739)
CryptozombiesHQ/cryptozombie-lessons: en/21/00-overview.md

 - MultiversX WASM VM: This fast and secure virtual machine executes smart contracts written in any programming language that can compile to WebAssembly, allowing for broader language support and greater flexibility.

 - High-performat and ultra-safe Rust framework: Rust is known to be a low-level multi-paradigm programing language featuring the same architecture as a computer processor, reason why it is designed for scripting high-performance and machine-efficient code. The framework of MultiversX provides super easy ways to design advanced smart contracts while at the same time keeping their size relatively low, keeping the gas costs low.  

MultiversX's innovative Adaptive State Sharding and Secure Proof of Stake consensus mechanism make it a great platform for deploying your CryptoZombies smart contract. By offering scalability, high performance, and developer-friendly tools, MultiversX can provide the ideal environment for your smart contract deployment.

Now that you're familiar with the exciting possibilities of MultiversX, let's get started with deploying your CryptoZombies smart contract on this groundbreaking platform!

## Installing mxpy

> Before starting it is recommended to install our tool **mxpy** and please make sure you have a working **Python 3** environment:

Reviewer: ```suggestion
Before starting this course, it is recommended that you install our CLI tool named **mxpy** . Also, please make sure you have a working **Python 3** environment:
```

- **3.8** or later on Linux and MacOS

In order to install **mxpy** using the `mxpy-up` installation script, run the following commands in a terminal:

```
wget -O mxpy-up.py https://raw.githubusercontent.com/multiversx/mx-sdk-py-cli/main/mxpy-up.py
python3 mxpy-up.py
```

The command above will install **mxpy**. Make sure you follow the instructions provided by the installer.

> For forther information please make sure to check https://docs.multiversx.com/sdk-and-tools/sdk-py/installing-mxpy/

Reviewer: ```suggestion
For further information please make sure to check https://docs.multiversx.com/sdk-and-tools/sdk-py/installing-mxpy/
```
